### ISSUE TITLE
scantailor-cli missing?

### ISSUE THREAD

**Issue by hanssep** [4lex4/scantailor-advanced#1]
This branch of scantailor is awesome and addresses some of the challenges I have faced when using scantailor. Much of the work we do is automated however and it seems scantailor-cli is missing from the release?

**Comment by 4lex4** [comment]
Yes, because it's need to be reworked to support the changes made and doesn't work correctly in the current state.

**Comment by mbrgm** [comment]
@4lex4 How much effort do you think this would be? Is it simply remapping parameters to CLI options and creating new ones for the additional features?

**Comment by M3ssman** [comment]
Hello,
I miss this API too. Are there any further plans regarding this issue?

**Comment by AurelioPuente** [comment]
Hey there, I'm going to tack on a request to this CLI thread. Any plans to add in automated processing with the defaults via CLI? Ideally with a watch folder. 

I am currently running PypdfOCR to OCR our completed Scan Tailored files. Ideally, I would like to have a folder that runs a watch command that converts the PDF to a ScanTailored PDF file with defaults and then spits it back out to PypdfOCR for OCRing.

**Comment by photosbyjeremy** [comment]
I suggest running the CLI based on the .ini config file settings and overwriting those with anything found in a .SCANTAILOR project file. Also, be able to save the settings that were run into a .SCANTAILOR file so it can be opened and adjusted if necessary. We’re currently planning to use AutoHotKey to do batch processing of a directory of scanned book directories with ScanTailor, but that’s not as easily scriptable or portable as a CLI.

Sent from a mobile device.
— — — — — — — — — —
Jeremy D Moore, MFA
alt.photosbyjeremy@gmail.com

________________________________
